library/src/main/java/com/cube/storm/ui/fragment/StormBottomTabsFragment.java

```diff
@@ -202,7 +202,7 @@ private void setTabItemContentDescriptions()
 			View tab = bottomNavigation.getViewAtPosition(bottomTabIdx);
 			if (tab != null)
 			{
-				String formatString = bottomNavigation.getCurrentItem() == bottomTabIdx ? getString(R.string.bottom_navigation_tab_selected) : getString(R.string.bottom_navigation_tab);
+				String formatString = selectedTab == bottomTabIdx ? getString(R.string.bottom_navigation_tab_selected) : getString(R.string.bottom_navigation_tab);
```

Comment:
is selectedTab the right index when the app starts? (without changing tab) :) 

also when opening a screen on another tab and coming back to the main screen

Reply:
It fails, so I need to restore the functionality, and make another PR